xds/src/main/java/io/grpc/xds/XdsAttributes.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import io.envoyproxy.envoy.api.v2.auth.SdsSecretConfig;
+import io.grpc.Attributes;
+import io.grpc.Grpc;
+
+import java.util.Map;
+
+/**
+ * Special attributes that are only useful to gRPC in the XDS context
+ */
+public final class XdsAttributes {
```

Comment:
Is it going to be used inside `io.grpc.xds` package only? If we can make it package private, otherwise we should annotate `@ExperimentalApi` and add `@since` in javadoc.

Reply:
Typically the XDS LB will set this on a subchannel, and that code most probably will go in this package. I'll make it package private (unless the preference is for `@ExperimentalApi`)